services-marionette/firefox_services_tests/apps/base.py

```diff
@@ -80,6 +81,9 @@ def click_element(self, by, locator):
     def send_keys_to_element(self, by, locator, string):
         self.marionette.find_element(by, locator).send_keys(string)
 
+    def wait(self, time):
+        self.action.wait(time).perform()
```

Comment:
If that is the only reason why you import `Actions` you should better utilize the `Wait(marionette).until()` pattern here. Also waiting for a hard-coded time is always a bad idea. It's always better to wait for a wanted condition.